styles.css

```diff
@@ -388,3 +391,149 @@ form button {
   border: none;
   background: white;
 }
+
+.dnav-list {
+  display: flex;
+}
+
+.detailed-nav {
+  display: none;
+}
+
+.dnav-list a {
+  text-decoration: none;
+  color: black;
+  margin-right: 32px;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.dnav-list a:hover {
+  text-decoration: underline;
+}
+
+@media screen and (min-width: 768px) {
+  /** Desktop styling **/
+  .li-class2 {
+    display: none;
+  }
+
+  .detailed-nav {
+    display: block;
+  }
+
+  .section2-container {
+    background-image: url(svgs/Header-desktop.svg);
+    padding: 100px 260px 259px 260px;
+  }
+
+  .c-class {
+    display: grid;
+    grid-template-columns: 48.5% 48.5%;
+    grid-gap: 3%;
+    border-radius: 24px;
+    padding: 24px;
+    width: 100%;
+    height: 100%;
+    margin-bottom: 142px;
+  }
```

Comment:
- Please, kindly ensure you've reversed the second and fourth card content to match the design from the Figma as much as you can. Since you've used grid for the layout, you can use the 
`grid-auto-flow: dense;
  direction: rtl;` or you can create a class and add to the second and last card in the media query since all cards uses the same class name and as you don't want to reverse all the cards.

```
.reverse{
  grid-auto-flow: dense;
  direction: rtl;
}
```

Kindly refer to the design template on Figma to see the cards to be reversed and after reversing the cards, please adjust your styling to match the Figma design as much as you can.

Reply:
Alright sure.